.docker/gs64/docker-tests.sh

```diff
@@ -144,13 +154,25 @@ assertOutputIncludesMessage "\[ERROR\] \"Name\" parameter not provided. You must
 print_success " Missing name, OK"
 run_launchpad_gem launchpad start greeter
 assertOutputIncludesMessage "\[ERROR\] \"Name\" parameter not provided. You must provide one." err
+print_success " Missing name and title, OK"
```

Comment:
Why do you have an empty space in the begging of every `print_success`?

Reply:
It's a bit of indentation to group similar tests in this ugly an basic bash test script :see_no_evil: